recipes/bitshuffle/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "bitshuffle" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kiyo-masui/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c3f4461d8013e3d9db0d58defec77b143164652de505a1fba3df088eaa19be2f
+
+build:
+  noarch: python
+  number: 0
```

Comment:
If you can [you should use pip here](http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip).

Reply:
sounds good! thanks @chrisburr !